### PR TITLE
create directory if doesn't exist for submissions download

### PR DESCRIPTION
**Pull request by heshamsafi**
#68

### CODE REVIEW

File: lib/commands/submission.js [hunk 1]
```diff
@@ -96,6 +96,11 @@ function exportSubmission(argv, problem, cb) {
         submissionState,
         h.langToExt(submission.lang));
 
+    try {
+      fs.statSync(argv.outdir);
```
Comment (skygragon) [inline lib/commands/submission.js]: better to use fs.existsSync instead?

**Comment by skygragon** [comment]
also would appreciate if you can merge those 2 tiny commits into single one, e.g. you can do it by reseting or rebasing in your local git repo.

**Comment by heshamsafi** [comment]
done

File: lib/commands/submission.js [hunk 2]
```diff
@@ -96,6 +96,10 @@ function exportSubmission(argv, problem, cb) {
         submissionState,
         h.langToExt(submission.lang));
 
+    if (!fs.existsSync(argv.outdir)) {
+      fs.mkdirSync(argv.outdir);
```
Comment (skygragon) [inline lib/commands/submission.js]: unfortunately this won't create directories recursively, while I think it's good enough for now.

Comment (heshamsafi) [reply lib/commands/submission.js]: If you want that, I can implement it but I think there is no need for it.

**Pull request merged**